### ISSUE TITLE
Two errors reported while testing in the CALVIN environment.

### ISSUE THREAD

**Issue by cbj20** [nickgkan/3d_diffuser_actor#95]
Hi, I attempted to test your model in CALVIN following the tutorial in README.md. However, I encountered the following two issues and I’m not sure if you have faced them before:

(1) In the section ‘Prepare data on CALVIN’, when running ‘python data_preprocessing/package_calvin.py --split training’, the process shuts down after handling the 494th data item with the following error:
Processing 494/17870,start id:1395669,end id:1395711 
Segmentation fault(core dumped)

(2) Considering that there were no errors in parsing the validation set, I proceeded to ‘Evaluate the pre-trained weights’ using the script you provided. However, I observed that the ‘rgb_obs’ data remains constantly at 0, and the ‘pcd_obs’ is also a constant, which suggests there might be an issue with the env configuration. 
I have strictly followed your tutorial for the setup, and I’m wondering if you are aware of any possible reasons for this error.
Thank you!
